src/pages/Product/components/ColorButton/ColorButton.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import '../ColorButton/ColorButton.scss'
+
+const ColorButton = (props) => {
+
+  return (
+    <button className={`colorButton ${props.changeBackground}`} onClick={props.changeColor}/>
+  )
+}
```

Comment:
```suggestion
const ColorButton = ({changeBackground, chageColor}) => {

  return (
    <button className={`colorButton ${changeBackground}`} onClick={changeColor}/>
  )
}
```

지난 Wrapup 시간에 배웠던 객체 구조분해 할당 개념을 적용하면
위와 같이 적용할 수 있겠네요!